promptsource/templates/guardian_authorship/cross_genre_1/templates.yaml

```
@@ -36,49 +44,133 @@ templates:
     jinja: "Generate an article on the topic of {{[\n  \"Politics\",\n  \"Society\"\
       ,\n  \"UK\",\n  \"World\",\n  \"Books\"\n][topic] }} |||\n{{article}}"
     metadata: !TemplateMetadata
-      choices_in_prompt: null
-      metrics: []
-      original_task: null
+      choices_in_prompt: false
+      metrics:
+      - BLEU
+      - ROUGE
+      original_task: false
     name: article_from_topic
     reference: ''
+  45e1aa31-6e76-4072-a6ee-ba99785999fc: !Template
+    answer_choices: catherinebennett ||| georgemonbiot ||| hugoyoung ||| jonathanfreedland
+      ||| martinkettle ||| maryriddell ||| nickcohen ||| peterpreston ||| pollytoynbee
+      ||| royhattersley ||| simonhoggart ||| willhutton ||| zoewilliams
+    id: 45e1aa31-6e76-4072-a6ee-ba99785999fc
+    jinja: 'Given the answer options of {% for author in answer_choices[:-1] %} {{author.capitalize()}},
+      {% endfor %} and {{answer_choices[-1].capitalize()}}, identify the author of
+      the passage, which is related to {{ ["Politics", "Society", "UK", "World", "Books"][topic]
+      }}, below.
+
+
+      {{article}}
+
+      |||
+
+      {{ answer_choices[author].capitalize() }}'
+    metadata: !TemplateMetadata
+      choices_in_prompt: false
+      metrics:
+      - Accuracy
+      original_task: true
+    name: who_wrote_article_with_topic_affirmative
+    reference: ''
   68bfa6a4-a89c-4be2-aa0b-cce1103e3ecf: !Template
-    answer_choices: null
+    answer_choices: Politics ||| Society ||| UK ||| World ||| Books
     id: 68bfa6a4-a89c-4be2-aa0b-cce1103e3ecf
-    jinja: "What is the topic of this article?\n\n{{article}} |||\n{{\n[\n  \"Politics\"\
-      ,\n  \"Society\",\n  \"UK\",\n  \"World\",\n  \"Books\"\n][topic]\n}}"
+    jinja: "What is the topic of this article? The answer options are {% for topic\
+      \ in answer_choices[:-1] %} {{topic.capitalize()}}, {% endfor %} {{answer_choices[-1].capitalize()}}.\n\
+      \n{{article}} \n|||\n{{answer_choices[topic]}}"
```

```suggestion
    jinja: "What is the topic of this article? The answer options are {{answer_choices|join(\"\
      , \")}}.\n\n{{article}} \n|||\n{{answer_choices[topic]}}"
```
simpler way to write it

---

Can be applied to a lot of other places